src/style.css

```diff
@@ -1 +1,76 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  --color-hover: #00acdc;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background: #e5e5e5;
+  height: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  height: 60px;
```

Comment:
if you didn't understand some comment - please, ask about it in the slack chat, instead of leaving things as they are

comment was: don't use height here, use paddings instead